src/server/player.js

```diff
@@ -150,7 +150,7 @@ export default class Player extends EventEmitter {
     this._changeState(STATES.BET_DONE);
     this.emit('raise', {
       id: this.id,
-      amount: amount
+      amount
```

Comment:
'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).